tests/unit/client/data-access/UlsLanguageTranslationRepository.spec.ts

```diff
@@ -0,0 +1,28 @@
+import UlsLanguageTranslationRepository from '@/client/data-access/UlsLanguageTranslationRepository';
+import { StringTMap } from '@/datamodel/LanguageTranslations';
+import { WikibaseContentLanguages } from '@/client/MwWindow';
+
+class MockWbWikibaseContentLanguages {
+	private translations: StringTMap<string>;
+	constructor( translations: StringTMap<string> ) {
+		this.translations = translations;
+	}
+
+	public getAllPairs(): StringTMap<string> {
+		return this.translations;
+	}
+}
+
+describe( 'UlsLanguageTranslationRepository', () => {
+	const langTranslation = { en: 'English' };
```

Comment:
These two variables `langTranslation` and `translator` should move into the `it`.